### PR TITLE
dissect: add module packages

### PR DESCRIPTION
**Pull request by D3vil0p3r**
Close https://github.com/BlackArch/blackarch/issues/3688

### CODE REVIEW

**Review by noraj — CHANGES_REQUESTED** [review]
That's a lot of work. I don't have time to test this right now. At a glance I can see license identifiers are old, could you upgrade to SPDX identifier please?

I hope in the long run that won't turn to be a hell to maintain and that we won't need to switch to virtualenv.

**Comment by D3vil0p3r** [comment]
Licenses upgraded. I did basic tests on all of them during packaging and they worked correctly.

**Comment by noraj** [comment]
> Licenses upgraded. I did basic tests on all of them during packaging and they worked correctly.

Yes but it will be soon obsolete when [AL RFC-0016](https://rfc.archlinux.page/0016-spdx-license-identifiers/) will be finalised.

https://wiki.archlinux.org/title/PKGBUILD#license

> Note: Arch Linux is [currently undergoing](https://rfc.archlinux.page/0016-spdx-license-identifiers/) a transition to SPDX style identifiers. Some issues may still require ironing out. This section focuses on the SPDX format. If you encounter any trouble, please report them and discuss with the community.

**Comment by D3vil0p3r** [comment]
At least we can manage that massive license change by 1 or 2 sed commands. Btw thanks for the last link.

File: packages/python-acquire/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+pkgname=python-acquire
+_pkgname=${pkgname#python-}
+pkgver=3.17.r0.gef2e3d1
+pkgrel=1
+pkgdesc='Quickly gather forensic artifacts from disk images or a live system into a lightweight container.'
+arch=('any')
+url='https://github.com/fox-it/acquire'
+license=('AGPL-3.0-or-later')
+depends=('python' 'python-dissect.cstruct' 'python-dissect.target' 'python-minio' 'python-pycryptodome' 'python-requests' 'python-requests-toolbelt' 'python-rich')
```
[the remark this answers is not in the document]
Comment (D3vil0p3r) [reply packages/python-acquire/PKGBUILD]: Fixed, thanks

File: packages/python-acquire/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  python -m build --wheel --no-isolation
+}
+
+package() {
+  cd $_pkgname
+
+  python -m installer --destdir="$pkgdir" dist/*.whl
+}
+
+check() {
```
[the remark this answers is not in the document]
Comment (D3vil0p3r) [reply packages/python-acquire/PKGBUILD]: Fixed

**Comment by noptrix** [comment]
@D3vil0p3r wrong arch setups :/ i've fixed them - please check my last commits for `dissect`. again, as soon as something gets compiled to a binary, we need to use `('x86_64', 'aarch64')`, otherwise leave as `('any')`

**Comment by D3vil0p3r** [comment]
Sorry @noptrix during the building of them I noted that it produced `bdist-x86_64` strings.. so I assumed all of them needed `('x86_64', 'aarch64')`. How did you check that, instead, `any` was needed for those dissect in the last commits?

**Comment by noptrix** [comment]
@D3vil0p3r no worries at all, just for the future... uhm, i am building them and see that nothign gets compiled, no binaries built. maybe there is another way to do that easier...

**Pull request merged**